addon/components/polaris-drop-zone.js

```diff
@@ -356,11 +356,13 @@ export default Component.extend(ContextBoundEventListenersMixin, {
       onDrop,
       onDropAccepted,
       onDropRejected,
+      state,
     } = this.getProperties(
       'disabled',
       'onDrop',
       'onDropAccepted',
-      'onDropRejected'
+      'onDropRejected',
+      'state'
     );
     if (disabled) {
```

Comment:
should this be something like
`if (disabled || (!allowMultiple && numFiles > 0)) {`

Reply:
Yes you're right... GitHub somehow had those changes collapsed so I missed them 🤦‍♂️